services/QuillLMS/config/routes.rb

```diff
@@ -595,6 +595,11 @@
     post '/learn_worlds/courses', to: 'learn_worlds#courses'
   end
 
+  namespace :canvas_integration do
+    get 'lti/launch_config.xml' => 'lti#launch_config'
```

Comment:
My first thought was "there's got to be a cleaner way to restrict a route to a single format", but then I looked and there's not.  TIL!

Reply:
I believe you can do this with [constraints](https://guides.rubyonrails.org/routing.html#request-based-constraints) if you want to, e.g.

```ruby
get 'lti/launch_config', constraints: lambda { |req| req.format == :xml }

Reply:
Another way is to use: `get '/lti/launch_config', to: 'lti#launch_config', constraints: { format: 'xml' }` but I don't know if that's any better.